awake.lua

```diff
@@ -299,34 +299,52 @@ function init()
   params:add{type = "trigger", id = "reset", name = "reset",
     action = function() reset() end}
   
-  params:add_group("synth",6)
+  params:add_group("synth", 7)
   cs_AMP = controlspec.new(0,1,'lin',0,0.5,'')
   params:add{type="control",id="amp",controlspec=cs_AMP,
     action=function(x) engine.amp(x) end}
 
   cs_PW = controlspec.new(0,100,'lin',0,50,'%')
   params:add{type="control",id="pw",controlspec=cs_PW,
-    action=function(x) engine.pw(x/100) end}
+    action=function(x) engine.pw(x/100); arcredraw() end}
```

Comment:
The redraw in the `step` function is pretty sparse, it goes as fast (or as slow) as the musical clock. If you change those params via Norns UI or via OSC, this looked bad as it was refreshing too rarely.

Putting the redraw here makes sure that any param:set and param:delta is immediately shown on the device, which looks entirely smooth. And thanks to putting this in the action, we don't have to worry about covering the bases manually: params set via UI, via Arc encoder, and via OSC are all refreshing Arc.